### PR TITLE
yak-shaving

### PR DESCRIPTION
**Pull request by BFGriffith**

### CODE REVIEW

**Comment by al-the-x** [comment]
Please put your checklist for this part of the assignment in the description of your PR.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+dependencies
```
Comment (al-the-x) [inline .gitignore:1]: I've seen this in your `.gitignore` files a couple of times, which leads me to believe you have a misunderstanding. There's nothing magical about the word `dependencies` when placed in your `.gitignore` file. Unless you have a directory or file somewhere in your project named `dependencies`, this does nothing.

Your project contains two sets of potential dependencies: one from Bower, another from NPM. Your `bower.json` file enumerates the dependencies that your project relies on in the browser, and the `bower` command installs these pacakges into the `bower_components/` directory. The `npm` command similarly installs packages into the `node_modules/` directory for use on the command line or in Node JS scripts. You might also be tempted to download packages from the internet _outside_ of these mechanisms and use them in your project. These would _also_ constitute dependencies for your project, but they're harder to track.

In avoiding [dependency hell](http://en.m.wikipedia.org/wiki/Dependency_hell), the Internets has given us [package managers](http://en.m.wikipedia.org/wiki/Package_manager) with manifest files like `bower.json` and `package.json` to track the specific version of a library or software package that we intend to use in our projects.

File: octocat [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Access-Control-Expose-Headers: ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
+Access-Control-Allow-Origin: *
+X-GitHub-Request-Id: 472A374E:7BA4:86EBA58:556DA36D
+Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
+X-Content-Type-Options: nosniff
+
+{
+  "message": "Not Found",
+  "documentation_url": "https://developer.github.com/v3"
+}
```
Comment (al-the-x) [inline octocat:24]: Probably not something you intended to include. Be careful of `git add .`!

**Comment by BFGriffith** [comment]
Yeah. I learned a lot about how .gitignore is supposed to work after we went over it in class and I totally realize the mistake I was making and will gradually work to correct that in my repos and branches. I think I have a much better sense now going forward about what I should and should not be ignoring and how to do that. Thanks!

**Pull request merged**